vc-verifier/kotlin/vcverifier/src/test/java/io/mosip/vercred/vcverifier/PresentationVerifierTest.kt

```diff
@@ -105,4 +112,78 @@ class PresentationVerifierTest {
         assertEquals(VPVerificationStatus.INVALID,verificationResult.proofVerificationStatus)
 
     }
+
+    @Test
+    @Timeout(20, unit = TimeUnit.SECONDS)
+    fun `should verify VP and return StatusList for revoked VC`() {
```

Comment:
here VP verification will fail since proof is not regenerated, so rename to `should verify VC and return VC status as revoked`